src/containers/Home/Search/index.test.tsx

```diff
@@ -0,0 +1,27 @@
+import {render,screen} from "@testing-library/react";
+import Search from "./index";
+import axios from "axios";
+test('renders inital document', () => {
+    render(<Search/>);
+    const divEle = screen.getByTestId("jobTest");
```

Comment:
- don't use `divEle`, should put name clear meaning, ex: `jobElement`
- And you should test all the Search bars, not only the `h2` element, maybe you can put the test-id in the bigger scope

Reply:
i've changed the name 
@nqbinh13